.buildkite/pipeline.yml

```diff
@@ -74,19 +74,20 @@ steps:
       - step: unit-tests-linux
         allow_failure: false
 
-  - label: ":windows: Run stack tests"
-    key: stack-tests-windows
-    command: ".buildkite/scripts/stack_tests_windows.ps1"
-    agents:
-      provider: "gcp"
-      image: "${WINDOWS_AGENT_IMAGE}"
-    depends_on:
-      - step: check-static
-        allow_failure: false
-      - step: unit-tests-windows
-        allow_failure: false
-    artifact_paths:
-      - 'C:\ProgramData\chocolatey\logs\chocolatey.log'
+  # Disable step due to docker errors: https://buildkite.com/elastic/elastic-package/builds/4023#0191ff3a-f18b-4e43-9219-4957da065c37/1827-1829
```

Comment:
Lets ensure we have an open issue to keep track of the problems with the docker installation.

Reply:
Created issue for this and added into the comment too: https://github.com/elastic/elastic-package/issues/2107